src/main/types.js

```diff
@@ -43,3 +43,15 @@ export type InflatedBlock = {
   compressedLength: number;
   buffer: ArrayBuffer;
 }
+
+// Coverage Track
+export type BinSummary = {
```

Comment:
Looks like these definitions snuck in after all. @armish Mind sending a quick PR to remove them?